sdk/servicebus/service-bus/test/internal/unit/sender.spec.ts

```diff
@@ -120,6 +120,20 @@ describe("sender unit tests", () => {
     });
   });
 
+  it("doesn't allow invalid enqueue time", async function() {
```

Comment:
interesting, how did you find the issue?

Reply:
#27368